bin/functions.d/eXist-settings.sh

```diff
@@ -76,7 +76,7 @@ restore_library_path() {
 
 set_client_java_options() {
     if [ -z "${CLIENT_JAVA_OPTIONS}" ]; then
-	CLIENT_JAVA_OPTIONS="-Xms128m -Xmx1024m -Dfile.encoding=UTF-8";
+	CLIENT_JAVA_OPTIONS="-Xms128m -Xmx2048m -Dfile.encoding=UTF-8";
```

Comment:
I don't think the Java Client needs very much at all (unless running in embedded mode), maybe just change the server setting?